lib/shopify_cli/theme/theme.rb

```diff
@@ -35,8 +35,10 @@ def json_files
         glob("**/*.json")
       end
 
-      def glob(pattern)
-        root.glob(pattern).map { |path| File.new(path, root) }
+      def glob(pattern, reject_dir: false)
+        root.glob(pattern).
```

Comment:
I don't really care. I have a mild preference for `.` at the end because it lets you copypaste into the terminal, but it seems our Rubocop is configured to want the beginning-of-line option, so I'll change it happily. I recall that [standard.rb went that way, too](https://github.com/testdouble/standard/issues/75).

Reply:
If you think `.` is more convenient that sounds good. I don't have very strong opinions here.